packages/framework/src/Model/Product/Search/ProductElasticsearchConverter.php

```diff
@@ -14,6 +14,7 @@ public function fillEmptyFields(array $product): array
     {
         $result = $product;
 
+        $result['id'] = $product['id'] ?? 0;
```

Comment:
what about this? :slightly_smiling_face: 
```suggestion
        $result['id'] = $product['id'] ?? $product['_id'] ?? 0;
```